server/routes/test_routes.ts

```diff
@@ -62,6 +62,7 @@ export function defineTestRoutes(router: IRouter,
         }
       });
     } catch (error) {
+      console.log(error);
```

Comment:
does error here contains username, password?

btw, do we still need a test_routes.ts?

Reply:
this API is for my local testing, will remove this route after the login page is complete